101-data-share-account/README.md

```diff
@@ -0,0 +1,24 @@
+# Copy data from one folder to another folder in an Azure Blob Storage
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-data-share-account/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-data-share-account/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-data-share-account/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-data-share-account/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-data-share-account/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-data-share-account/CredScanResult.svg)
+
+[![Deploy to Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-data-share-account%2Fazuredeploy.json)
+
+[![Deploy to AzureGov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-data-share-account%2Fazuredeploy.json)
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-data-share-account%2Fazuredeploy.json)
+
+When you deploy this Azure Resource Manager template, a data share account is created.
```

Comment:
what is a data share account?  is there a doc link?

Reply:
https://docs.microsoft.com/en-us/azure/data-share/overview.  I've add this link to the descriptions. should I apply this anywhere else?

Reply:
> https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?



> what is a data share account? is there a doc link?

https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?

Reply:
https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?